src/templates/build-page.js

```diff
@@ -18,46 +18,55 @@ export const BuildPageTemplate = ({
   ajaxRequestStates = {},
 }) => {
   return (
-    <div className="main-content">
+    <div className="main-content build">
```

Comment:
@maymax777, the class name should be `build-content`. and it can be used as a root block in `build.scss`.